### ISSUE TITLE
GitHub Issue editor : user hostile behaviour

### ISSUE THREAD

**Issue by jmdh01** [brave/browser-laptop#1941]
**Did you search for similar issues before submitting this one?**

**Describe the issue you encountered:**

**Expected behavior:**

- Platform (Win7, 8, 10? OS X? Linux distro?):

- Brave Version:

- Steps to reproduce:
    1.
    2.
    3.

- Screenshot if needed:

- Any related issues:
Issue #1940 was arbitrarily terminated and closed without clicking on the "Submit new issue" button. The submission was being typed in Brave v0.10.0 , Ubuntu 15.10, Trinity

**Comment by bbondy** [comment]
If there's an issue with github then you'll have to report it to github. #1940 shows that you closed the issue in the log.  This repo is for bugs with the Brave browser specifically not with github.

**Comment by jmdh01** [comment]
I was typing normally (ie one finger style) and something, possible 
brave, just closed the dialog without any warning whatsoever. That is 
why I raised the issue. I had just completed the 'title' box and had 
just started to type the body (two characters at most), when bang, I was 
looking at the issues list, which did not contain #1940. Eventually, I 
found that it had been closed, and therefore would not be listed. 
Bearing in mind that github is more mature than brave, brave came under 
suspicion. John Hunter (jmdh01).
